system/settings/settings.xml

```diff
@@ -2544,7 +2548,7 @@
           <level>2</level>
           <default>false</default>
           <dependencies>
-            <dependency type="visible">
+            <dependency type="enable">
```

Comment:
I'm not sure if you take into consideration the number of totally clueless users there are. To my mind the GUI settings should guide users to the correct conclusion, which I believe greying out and not hiding does.

User connected to TV that does not support passthrough, clueless user then posts why do I not see any passthrough options..

User connected to device with only Dolby Digital support hence no DTS support (or any of the other formats), clueless user posts why do I not see the DTS option.

For these clueless users, showing the setting but greying it out gives a pretty big visual clue that it isn't supported.

But we can discuss more at Devcon if you disagree.